### PR TITLE
Fix french lab test concept names

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/lab_tests.csv [hunk 1]
```diff
@@ -98,17 +105,17 @@ e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,M
...
+faabf960-2025-42e1-8eed-f7825cfa6e2d,,CIEL:162614,Vaginal Swab,Écouvillon vaginal,Vaginal Swab,Écouvillon vaginal,,,Sample,N/A,,,Gram-positive Cocci;Gram-negative Cocci;Gram-positive Diplococci;Gram-negative Diplococci;Gram-negative Intracellular Diplococci;Gram-positive Bacilli;Gram-negative Bacilli;Coccobacilli;Vaginal Swab (Gram Stain);Swab White Blood Cells;Swab Red Blood Cells;Swab Bacteria;Swab Epithelial Cells;Swab Budding Yeasts;Swab Simple Yeasts;Swab Mycelium;Swab Trichomonas vaginalis;Clue cell;Vaginal Swab (Wet Mount),,,,,,,,
+7d14260b-2000-4863-9d8b-71dcb3a32f36,,CIEL:161451,Stool,Tabouret,Stool,Tabouret,,,Sample,N/A,,,Stool Consistence;Stool Appearance;Stool Colour;Stool Parasites;Stool Analysis,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: "Stool" = "Selles"

File: configuration/concepts/lab_tests.csv [hunk 2]
```diff
@@ -98,17 +105,17 @@ e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,M
...
+09db95e4-185c-4a22-9464-10c77dff27be,,CIEL:159994,Urine,Urine,Urine,Urine,,,Sample,N/A,,,Gravindex;Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Urine Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Urine Protein;Urine White Blood Cells;Urine Red Blood Cells;Urine Bacteria;Urine Epithelial Cells;Urine Budding Yeasts;Urine Simple Yeasts;Urine Mycelium;Macrophages;Kidney Cells;Leukocyte Cylinders;Granular Cylinders;Hyaline Cylinders;Erythrocyte Cylinders;Crystals;Urine Trichomonas Vaginalis;Urinalysis,,,,,,,,
+faabf960-2025-42e1-8eed-f7825cfa6e2d,,CIEL:162614,Vaginal Swab,Écouvillon vaginal,Vaginal Swab,Écouvillon vaginal,,,Sample,N/A,,,Gram-positive Cocci;Gram-negative Cocci;Gram-positive Diplococci;Gram-negative Diplococci;Gram-negative Intracellular Diplococci;Gram-positive Bacilli;Gram-negative Bacilli;Coccobacilli;Vaginal Swab (Gram Stain);Swab White Blood Cells;Swab Red Blood Cells;Swab Bacteria;Swab Epithelial Cells;Swab Budding Yeasts;Swab Simple Yeasts;Swab Mycelium;Swab Trichomonas vaginalis;Clue cell;Vaginal Swab (Wet Mount),,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: "Prélèvement vaginal". That should replace "Ecouvillon" everywhere.

File: configuration/concepts/lab_tests.csv [hunk 3]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
-1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Epithelial Cells,Cellules épithéliales,Epithelial Cells,Cellules épithéliales,,,LabTest,Text,,,,,,,,,,,
-d2f224b3-7604-4ae0-a7ae-93ea92ce7106,,,Budding Yeasts,Levures bourgeonnates,Budding Yeasts,Levures bourgeonnates,,,LabTest,Text,,,,,,,,,,,
-8ac2c2c6-5333-43ac-b2ee-25077160b443,,,Simple Yeasts,Levures simples,Simple Yeasts,Levures simples,,,LabTest,Text,,,,,,,,,,,
-93cbd183-498b-4dd0-8002-b0d49d00709c,,,Mycelium,Miscelium,Mycelium,Miscelium,,,LabTest,Text,,,,,,,,,,,
+e3ae7371-8a91-413e-8f99-93bc8f45a07a,,,Urine White Blood Cells,Globules blancs urinaires,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+eceed5e9-c513-4ff0-b8a2-900f53599c41,,,Swab White Blood Cells,Écouvillon de globules blancs,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+668b296c-a1fc-4d2f-bfad-1997704a4762,,,Urine Red Blood Cells,Globules rouges urinaires,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
+d4708141-0562-49c4-aabe-b03a2dbf7c2b,,,Swab Red Blood Cells,Écouvillon de globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Prélèvement de globules rouges

File: configuration/concepts/lab_tests.csv [hunk 4]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
-668b296c-a1fc-4d2f-bfad-1997704a4762,,,Red Blood Cells,Globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
-168f4adf-4beb-49d2-8070-b2776ccd2b23,,,Bacteria,Bactéries,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
-1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Epithelial Cells,Cellules épithéliales,Epithelial Cells,Cellules épithéliales,,,LabTest,Text,,,,,,,,,,,
-d2f224b3-7604-4ae0-a7ae-93ea92ce7106,,,Budding Yeasts,Levures bourgeonnates,Budding Yeasts,Levures bourgeonnates,,,LabTest,Text,,,,,,,,,,,
-8ac2c2c6-5333-43ac-b2ee-25077160b443,,,Simple Yeasts,Levures simples,Simple Yeasts,Levures simples,,,LabTest,Text,,,,,,,,,,,
-93cbd183-498b-4dd0-8002-b0d49d00709c,,,Mycelium,Miscelium,Mycelium,Miscelium,,,LabTest,Text,,,,,,,,,,,
+e3ae7371-8a91-413e-8f99-93bc8f45a07a,,,Urine White Blood Cells,Globules blancs urinaires,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+eceed5e9-c513-4ff0-b8a2-900f53599c41,,,Swab White Blood Cells,Écouvillon de globules blancs,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+668b296c-a1fc-4d2f-bfad-1997704a4762,,,Urine Red Blood Cells,Globules rouges urinaires,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv:60]: Prélèvement de globules blancs

File: configuration/concepts/lab_tests.csv [hunk 5]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+d4708141-0562-49c4-aabe-b03a2dbf7c2b,,,Swab Red Blood Cells,Écouvillon de globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
+168f4adf-4beb-49d2-8070-b2776ccd2b23,,,Urine Bacteria,Bactéries urinaires,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
+02a4cfd9-196c-4ce7-90a8-c67f52c8fa7e,,,Swab Bacteria,Bactéries sur écouvillon,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Not sure if it should be "Prélèvement de bactéries" or "Bactéries sur prélèvement". Depends on what Swab Bacteria is really.

File: configuration/concepts/lab_tests.csv [hunk 6]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Urine Epithelial Cells,Cellules épithéliales d'urine,Epithelial Cells,Cellules épithéliales,,,LabTest,Text,,,,,,,,,,,
+b81d6efb-79d3-4c75-802f-b37e266eea10,,,Swab Epithelial Cells,Cellules épithéliales sur écouvillon,Epithelial Cells,Cellules épithéliales,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: I guess "Prélèvement de cellules épithéliales"

File: configuration/concepts/lab_tests.csv [hunk 7]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+0c6000c4-25db-4cf3-b25f-2664f601ef23,,,Urine Budding Yeasts,Levures bourgeonnantes d'urine,Budding Yeasts,Levures bourgeonnates,,,LabTest,Text,,,,,,,,,,,
+71e57af5-82de-48f1-9707-e4fab8cec9c9,,,Swab Budding Yeasts,Levures bourgeonnantes sur écouvillon,Budding Yeasts,Levures bourgeonnates,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Same here as above.

File: configuration/concepts/lab_tests.csv [hunk 8]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+8ac2c2c6-5333-43ac-b2ee-25077160b443,,,Urine Simple Yeasts,Levures simples d'urine,Simple Yeasts,Levures simples,,,LabTest,Text,,,,,,,,,,,
+ecada6af-564c-428f-ada6-c356e9efd8e5,,,Swab Simple Yeasts,Écouvillon de levures simples,Simple Yeasts,Levures simples,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Same

File: configuration/concepts/lab_tests.csv [hunk 9]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+93cbd183-498b-4dd0-8002-b0d49d00709c,,,Urine Mycelium,Mycélium urinaire,Mycelium,Mycélium,,,LabTest,Text,,,,,,,,,,,
+a2288ed4-c0fc-44f0-bd97-156a7f9a6fb7,,,Swab Mycelium,Écouvillon de mycélium,Mycelium,Mycélium,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Same

File: configuration/concepts/lab_tests.csv [hunk 10]
```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
+6a5a0914-a34d-4d5c-954f-020dd925596c,,CIEL:163648,Urine Trichomonas Vaginalis,Urine Trichomonas vaginalis,Trichomonas Vaginalis,Trichomonas vaginalis,,,LabTest,Text,,,,,,,,,,,
+885869cb-6bef-4e7e-bb37-9719b6d82e7b,,CIEL:163648,Swab Trichomonas Vaginalis,Écouvillon trichomonas vaginalis,Trichomonas Vaginalis,Trichomonas vaginalis,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Same

**Review by rbuisson — CHANGES_REQUESTED** [review]
Thanks.
Some changes in fr required.
And I couldn't find "Selles consistance" as per your screenshot. But that should be: "Consistance des selles"

File: configuration/concepts/lab_tests.csv [hunk 11]
```diff
@@ -98,17 +105,17 @@ e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,M
...
+1eb4764c-91dd-4964-813f-e4eafacf7e0c,,,Gyn Department,Département Gyn,Gyn Department,Département Gyn,,,Department,N/A,,,Gram-positive Cocci;Gram-negative Cocci;Gram-positive Diplococci;Gram-negative Diplococci;Gram-negative Intracellular Diplococci;Gram-positive Bacilli;Gram-negative Bacilli;Coccobacilli;Vaginal Swab (Gram Stain);Swab White Blood Cells;Swab Red Blood Cells;Swab Bacteria;Swab Epithelial Cells;Swab Budding Yeasts;Swab Simple Yeasts;Swab Mycelium;Swab Trichomonas vaginalis;Clue cell;Vaginal Swab (Wet Mount),,,,,,,,
+c7dc14b7-ee8f-4a4f-8a5f-863baf787a24,,,Stool Department,Département de tabouret,Stool Department,Département de tabouret,,,Department,N/A,,,Stool Consistence;Stool Appearance;Stool Colour;Stool Parasites;Stool Analysis,,,,,,,,
+5a0ae1f0-dac8-4df9-b5b7-59d16cc6ad03,,,Lab Departments,Départements de laboratoire,Lab Departments,Départements de laboratoire,,,ConvSet,N/A,,,Blood Department;Urine Department;Gyn Department;Stool Department,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: "Service analyses sanguines"
"Service analyses d'urine"
"Service analyses gynécologiques"
"Service analyses des selles"
"Services laboratoire"

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv:60]: Done

File: configuration/concepts/lab_tests.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 7 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 9 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 10 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

**Pull request merged**